### PR TITLE
Options documentation

### PR DESCRIPTION
**Pull request by soullivaneuh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,3 +39,27 @@ composer require sllh/composer-lint
 That's it! Composer will enable automatically the plugin as soon it's installed.
 
 Just run `composer validate` command to see the plugin working.
+
+## Configuration
+
+You can configure the plugin via the [`COMPOSER_HOME/config.json`](https://getcomposer.org/doc/03-cli.md#composer-home) file. Here is the default one:
```
Comment (greg0ire) [inline README.md:45]: ~~`$COMPOSER_HOME` ?~~ the doc doesn't use a dollar sign, so …

File: README.md [hunk 2]
```diff
@@ -39,3 +39,27 @@ composer require sllh/composer-lint
...
+            "type": true,
+            "minimum-stability": true,
+            "version-constraints": true
+        },
+        "sort-packages": false
+    }
+}
+```
+
+* `php`: Checks if PHP requirement is set on the `require` section.
```
Comment (greg0ire) [inline README.md]: "the PHP requirement"

File: README.md [hunk 3]
```diff
@@ -39,3 +39,27 @@ composer require sllh/composer-lint
...
+* `type`: Check if package `type` is defined.
```
Comment (greg0ire) [inline README.md]: Checks

File: README.md [hunk 4]
```diff
@@ -39,3 +39,27 @@ composer require sllh/composer-lint
...
+* `minimum-stability`: Checks if `minimum-stability` is set. It raise an error if yes, except for `project` packages.
```
Comment (greg0ire) [inline README.md]: - "raises"
- "if yes" => "if it is"

**Comment by soullivaneuh** [comment]
Thank you @greg0ire for the review. :+1:

**Pull request merged**